core/src/main/scala/org/apache/spark/api/python/PythonRDD.scala

```diff
@@ -83,10 +83,23 @@ private[spark] case class PythonFunction(
  */
 private[spark] case class ChainedPythonFunctions(funcs: Seq[PythonFunction])
 
+/**
+ * Enumerate the type of command that will be sent to the Python worker
+ */
+private[spark] object PythonEvalType {
+  val NON_UDF = 0
+  val SQL_BATCHED_UDF = 1
+  val SQL_ARROW_UDF = 2
```

Comment:
the new udf parameter is pandas `Series`, I think it's more accurate to call it `SQL_PANDAS_UDF`.